### PR TITLE
Show last delta migration in publication log

### PR DESCRIPTION
**Pull request by HPrinz**
RISDEV-4294

### CODE REVIEW

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/OriginalXmlDTO.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Entity
+@Table(name = "original_xml", schema = "incremental_migration")
+public class OriginalXmlDTO {
```
Comment (kiwikern) [inline backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/OriginalXmlDTO.java:27]: question: Why OriginalXmlDTO and not MigrationDTO?

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/Migration.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package de.bund.digitalservice.ris.caselaw.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+import lombok.Builder;
+
+@Builder(toBuilder = true)
+public record Migration(
+    String xml,
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY, value = "date") Instant migratedDate)
```
Comment (kiwikern) [inline backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/Migration.java:10]: question: When/how is this data saved?

File: frontend/src/components/PublicationDocument.vue [hunk 3]
```diff
@@ -120,6 +120,19 @@ function getMissingEnsuingDecisionFields(ensuingDecision: EnsuingDecision) {
...
+function getHeader(item: XmlMail) {
+  switch (item.type) {
+    case PublicationHistoryRecordType.PUBLICATION_REPORT:
+      return "Juris Protokoll - " + item.date
+    case PublicationHistoryRecordType.PUBLICATION:
+      return "Xml Email Abgabe - " + item.date
+    case PublicationHistoryRecordType.MIGRATION:
+      return "Letzte Delta Migration - " + item.date
+    default:
+      return "Unknown Type - " + item.date
```
[the remark this answers is not in the document]
Comment (HPrinz) [reply frontend/src/components/PublicationDocument.vue:132]: Good catch, I'll adjust this - thank you!

File: frontend/src/components/PublicationDocument.vue [hunk 4]
```diff
@@ -388,6 +397,12 @@ const fieldsMissing = computed(() => {
               <div class="ds-label-section text-gray-900">ALS</div>
               <CodeSnippet v-if="!!item?.xml" title="XML" :xml="item.xml" />
             </div>
+            <div
+              v-if="item.type == PublicationHistoryRecordType.MIGRATION"
```
[the remark this answers is not in the document]
Comment (HPrinz) [reply frontend/src/components/PublicationDocument.vue:401]: Yes, I agree 👍 . Will fix it

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/OriginalXmlDTO.java (hunk 1 above)
Comment (HPrinz) [reply backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/OriginalXmlDTO.java:27]: Good question! We name the database entities after the table names (`original_xml`)  and not the domain models

**Pull request merged**